src/components/enterprise/dashboard/sidebar/DashboardSidebar.jsx

```diff
@@ -32,19 +32,20 @@ class DashboardSidebar extends React.Component {
     ));
   }
 
-  renderLearningCoordinatorHelpText(enterpriseName, enterpriseEmail) {
+  renderLearningCoordinatorHelpText() {
+    const { pageContext: { enterpriseName, enterpriseEmail } } = this.context;
     if (enterpriseEmail) {
       return (
         <a href={`mailto:${enterpriseEmail}`}>
-          Contact your {enterpriseName} learning coordinator
+          contact your {enterpriseName} learning coordinator
```

Comment:
Now that this text is only rendered in one place, lowering it to a lowercase "c" as opposed to "C" so it fits well with the surrounding sentence.